### ISSUE TITLE
loss = HierarchicalSoftmaxLoss(root=self.metadata.root)

### ISSUE THREAD

**Issue by ZhangRan24** [rbturnbull/hierarchicalsoftmax#1]
I'm confused by this code：loss = HierarchicalSoftmaxLoss(root=self.metadata.root). How do I get the self.metadata.root.
 I would appreciate it if you could provide specific instructions for the form self.metadata.root

**Comment by rbturnbull** [comment]
Hi @ZhangRan24 - thanks for spotting this. Sorry, I made a mistake in the readme. This should just say ``loss = HierarchicalSoftmaxLoss(root=root)``. I've fixed the readme now. Please let me know if you have any other trouble.

**Comment by ZhangRan24** [comment]
Thank you very much for your correction and answer!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "rbturnbull/hierarchicalsoftmax"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月11日(星期天) 晚上8:12
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [rbturnbull/hierarchicalsoftmax] loss = HierarchicalSoftmaxLoss(root=self.metadata.root) (Issue #1)





 
Hi @ZhangRan24 - thanks for spotting this. Sorry, I made a mistake in the readme. This should just say loss = HierarchicalSoftmaxLoss(root=root). I've fixed the readme now. Please let me know if you have any other trouble.
